### ISSUE TITLE
Error on checkout 

### ISSUE THREAD

**Issue by dani97** [deity-io/falcon#422]
**Describe the bug**
A graphql error is displayed when tried to place order, 
GraphQL error: Please check the shipping address information. "regionId" is required. Enter and try again.

**To Reproduce**
1. Add some products to cart and proceed to checkout.
2. Fill the email address,shipping address and billing address
3. select shipping method and payment option, and place order
4. you get the error message
GraphQL error: Please check the shipping address information. "regionId" is required. Enter and try again.

**Expected behavior**
Region field should be present in addresses.

**Screenshots**
![image](https://user-images.githubusercontent.com/13298685/56356235-df22ea80-61f5-11e9-95c4-30eb5d4e178c.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 16.04
 - Browser chrome
 - Version 73

**Comment by mateuszwozniak** [comment]
@dani97 can you please check if that error still happens for you on https://demo.deity.io?

**Comment by dani97** [comment]
@mateuszwozniak yes still I get the error for both the guest user and registered user accounts.

**Comment by cane-lagerwaard** [comment]
I've looked into this issue and found out that it happens when you select a country outside of the EU. Probably this is caused by the shopping configuration from Magento itself. @sergy-gzh could you confirm?

Additionally: After you have pressed "place order" while i.e. India was selected as country, and you try to change the address to one that should work, it still doesn't work unless you refresh the page and fill in the form from scratch.

**Comment by sergy-gzh** [comment]
@cane-lagerwaard `regionId` is a required parameter for such countries as United States for example.
Falcon should submit this data with the rest of the address data.

**Comment by cane-lagerwaard** [comment]
@sergy-gzh I can see `regionId` does indeed not get sent in the payload, it is also not possible to enter it in the form itself
